numbat/tests/interpreter.rs

```diff
@@ -362,6 +362,7 @@ fn test_other_functions() {
     expect_output("floor(3.9)", "3");
     expect_output("round(3.9)", "4");
     expect_output("round(3.1)", "3");
+    expect_output("random() == random()", "false");
```

Comment:
This test could fail in theory (though very unlikely), so not sure we should have it.

Reply:
Actually this is probably so unlikely that we can probably disregard it entirely, /shrug